ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h

```diff
@@ -1216,6 +1219,7 @@ class TDqComputeActorBase : public NActors::TActorBootstrapped<TDerived>
         }
 
         ReportStateAndMaybeDie(ev->Get()->Record.GetStatusCode(), issues, true);
+        DoTerminateImpl();
```

Comment:
Should `DoTerminateImpl();` be also called here?
https://github.com/ydb-platform/ydb/pull/15742/files#diff-32f961501ccce63b12647403637de6cc06ea210fab609dff0a93506a157e8e7cR1198

Reply:
And here:
https://github.com/ydb-platform/ydb/pull/15742/files#diff-32f961501ccce63b12647403637de6cc06ea210fab609dff0a93506a157e8e7cR1148